### PR TITLE
Add roll docs, stop_prank docs

### LINKED ISSUE #272: Document `StartRoll` cheatcode

### PR DESCRIPTION
**Pull request by MaksymilianDemitraszek**
Closes #272 

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code

### CODE REVIEW

File: docs/src/appendix/forge-library/start_roll.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#[external(v0)]
+impl IRollChecker of super::IRollChecker<ContractState> {
+    fn get_block_number(ref self: ContractState) -> u64 {
+        starknet::get_block_info().unbox().block_number
+    }
+}
+// ...
+```
+
+We can use roll in a test to change
```
Comment (war-in) [inline docs/src/appendix/forge-library/start_roll.md:23]:
```suggestion
We can use `start_roll` to change the current block number inside a test
```

**Comment by MaksymilianDemitraszek** [comment]
@THenry14 @war-in Yea I should have re-read this, pushing 5 min before end of the day always end like this, sorry :p

**Review by piotmag769 — CHANGES_REQUESTED** [review]
Can we add it to the index though? It is not displayed anywhere xD

**Comment by cptartur** [comment]
Taking over this one

File: docs/src/appendix/cheatcodes/stop_prank.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# `stop_prank`
+
+> `fn stop_prank(contract_address: ContractAddress)`
+
+Cancels the [`stop_prank`](./stop_prank.md) for the contract at the given address.
```
[the remark this answers is not in the document]
Comment (piotmag769) [reply docs/src/appendix/cheatcodes/stop_prank.md:5]: It's a separate cheatcode

[the remark this answers is not in the document]
Comment (piotmag769) [reply docs/src/appendix/cheatcodes/stop_prank.md:5]: Yea, sure xD mb

File: docs/src/appendix/cheatcodes/start_prank.md [hunk 3]
```diff
@@ -2,18 +2,33 @@
 
 > `fn start_prank(contract_address: ContractAddress, caller_address: ContractAddress)`
 
-Changes the caller address in the `ExecutionInfo` struct and methods utilizing it (e.g. `get_caller_address`).
+Changes the caller address for a contract at the given address.
+This change can be canceled with [`stop_prank](./stop_prank.md).
```
Comment (piotmag769) [inline docs/src/appendix/cheatcodes/start_prank.md:6]: Missing "`"

File: docs/src/appendix/cheatcodes/start_roll.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- `contract_address` - target contract address
+- `block_number` - mocked value of block number
+
+For contract implementation:
+
+```rust
+// ...
+#[external(v0)]
+impl IContractImpl of IContract<ContractState> {
+    fn get_block_number(ref self: ContractState) -> u64 {
```
Comment (piotmag769) [inline docs/src/appendix/cheatcodes/start_roll.md:17]: getter doesn't require self to be mutable
```suggestion
    fn get_block_number(self: @ContractState) -> u64 {
```

File: docs/src/appendix/cheatcodes/stop_prank.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cptartur) [reply docs/src/appendix/cheatcodes/stop_prank.md]: Yes, fixed

File: docs/src/appendix/cheatcodes/start_prank.md [hunk 5]
```diff
@@ -2,18 +2,33 @@
...
+- `contract_address` - target contract address
 - `caller_address` - caller address to be set
 
+For contract implementation:
+
+```rust
+// ...
+#[external(v0)]
+impl IContractImpl of IContract<ContractState> {
+    fn get_caller_address( ref self: ContractState) -> ContractAddress {
```
Comment (piotmag769) [inline docs/src/appendix/cheatcodes/start_prank.md:17]: getter doesn't require self to be mutable
```suggestion
    fn get_caller_address(self: @ContractState) -> ContractAddress {
```

**Review by THenry14 — APPROVED** [review]

**Review by piotmag769 — APPROVED** [review]

**Pull request merged**